deploy/chart/ingressmonitorcontroller/templates/clusterrole.yaml

```diff
@@ -0,0 +1,81 @@
+{{- if and ( .Values.rbac.create ) (eq .Values.watchNamespace "") }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ template "name" . }}-cluster-role
+  labels:
+{{ include "labels" $ | indent 4 }}
+rules:
+- apiGroups:
+  - ""
+  - "extensions"
+  - "route.openshift.io"
+  resources:
+  - ingresses
+  - routes
+  - secrets
+  verbs:
+  - list
+  - get
+  - watch
+- apiGroups:
+  - ""
+  resources:
+  - events
+  - services
+  - configmaps
+  - services/finalizers
+  verbs:
+  - create
+  - delete
+  - get
+  - list
+  - patch
+  - update
+  - watch
+- apiGroups:
+  - apps
+  resources:
+  - deployments
+  - daemonsets
+  - replicasets
+  - statefulsets
+  verbs:
+  - "*"
```

Comment:
Are you sure we need to assign open access for these resources?

Reply:
just copied the role from `deploy/` folder, do you think if only `list,get,watch` will work?

Reply:
Yeah that should be it. I think `watch` won't be required now either.